src/editors/editorTestRender.tsx

```diff
@@ -23,20 +27,24 @@ export const editorRender = (
     initialState = {},
     learningContextId = 'course-v1:Org+COURSE+RUN',
     ...options
-  }: Omit<WrapperOptions, 'extraWrapper'> & { initialState?: PartialEditorState, learningContextId?: string } = {},
+  }: WrapperOptions & RouteOptions & { initialState?: PartialEditorState, learningContextId?: string } = {},
```

Comment:
```suggestion
  }: Omit<WrapperOptions, 'extraWrapper'> & RouteOptions & { initialState?: PartialEditorState, learningContextId?: string } = {},
```

We still need to omit `extraWrapper` here because this function sets `extraWrapper`, so any `extraWrapper` passed via `options` will be ignored.